docs/tutorials/podman_tutorial.md

```diff
@@ -129,7 +129,7 @@ $ sudo podman inspect -l | grep IPAddress\":
         "IPAddress": "10.88.6.140",
 ```
 
-Note: The -l is convenience arguement for **latest container**.  You can also use the container's ID instead
+Note: The -l is convenience argument for **latest container**.  You can also use the container's ID instead
```

Comment:
I know not your doing, but since you're in here, would you mind changing "is convenience" to "is a convenience"?  Otherwise LGTM

Reply:
@TomasTomecek Done :+1: